docs/client/_client-sdk-template.mdx

```diff
@@ -88,14 +88,18 @@ For example, if you have feature gates that should only pass for users with emai
 
 ## Statsig Options {#statsig-options}
 
-<div>{props.options ?? '(Coming soon)'}</div>
+<div>{props.options ?? "(Coming soon)"}</div>
 
 ## Shutting down Statsig
 
 In order to save users' data and battery usage, as well as prevent logged events from being dropped, we keep event logs in client cache and flush periodically. Because of this, some events may not have been sent when your app shuts down. To make sure all logged events are properly flushed or saved locally, you should tell Statsig to shutdown when your app is closing:
 
 <div>{props.shutdown}</div>
 
+## Reference
+
+<div>{props.reference ?? "(Coming soon)"}</div>
+
```

Comment:
Just add this when you add it to client SDKs - this diff doesnt add a reference to any client SDK